tests/integration/root.bats

```diff
@@ -11,7 +11,7 @@ function setup() {
 function teardown() {
 	if [ -n "$ALT_ROOT" ]; then
```

Comment:
Is it possible for `ALT_ROOT` to be unset before this function is called? :eyes:  (ie, if this function got called twice somehow?  from the name it sounds like that's terribly unlikely, but maybe worth thinking about :see_no_evil:)

Reply:
It used to be the case earlier that tests called teardown before setup. This must be a leftover from the era.

You're right that this does not work well with `set -u`. Let me remove it.

Reply:
Reworked ALT_ROOT handling in `teardown` is a separate commit; PTAL.